From ece7e5950dfc4b2ac015a3e399ee53183c9f5063 Mon Sep 17 00:00:00 2001
From: Jan Kratochvil <jan.kratochvil@redhat.com>
Date: Mon, 23 Apr 2012 17:26:00 +0000
Subject: [PATCH] gdb/doc/
 	* gdb.texinfo (Auto-loading safe path): Add trailing @dots{}.
 	Three times.

---
 gdb/doc/ChangeLog   |    3 +++
 gdb/doc/gdb.texinfo |    6 +++---
 2 files changed, 6 insertions(+), 3 deletions(-)

Index: gdb-7.2/gdb/doc/gdb.texinfo
===================================================================
--- gdb-7.2.orig/gdb/doc/gdb.texinfo
+++ gdb-7.2/gdb/doc/gdb.texinfo
@@ -20017,11 +20017,11 @@ Specify this trusted directory (or a fil
 You have to specify also any existing directories displayed by
 by @samp{show auto-load safe-path} (such as @samp{/usr:/bin} in this example).
 
-@item @kbd{gdb -iex "set auto-load safe-path /usr:/bin:~/src/gdb"}
+@item @kbd{gdb -iex "set auto-load safe-path /usr:/bin:~/src/gdb" @dots{}}
 Specify this directory as in the previous case but just for a single
 @value{GDBN} session.
 
-@item @kbd{gdb -iex "set auto-load safe-path"}
+@item @kbd{gdb -iex "set auto-load safe-path" @dots{}}
 Disable auto-loading safety for a single @value{GDBN} session.
 This assumes all the files you debug during this @value{GDBN} session will come
 from trusted sources.
@@ -20036,7 +20036,7 @@ On the other hand you can also explicitl
 also suppresses any such warning messages:
 
 @table @asis
-@item @kbd{gdb -iex "set auto-load no"}
+@item @kbd{gdb -iex "set auto-load no" @dots{}}
 You can use @value{GDBN} command-line option for a single @value{GDBN} session.
 
 @item @file{~/.gdbinit}: @samp{set auto-load no}
